src/lib/globalShortcutManager.ts

```diff
@@ -0,0 +1,129 @@
+import { Store, load as loadStore } from '@tauri-apps/plugin-store'
```

Comment:
We should use sqlite for the users persistent settings like a keyboard shortcut

Reply:
This file is nicely organized and largely fits the "repo" structure we use for accessing sqlite data. I would move this file to be part of `src/db/ebb` and probably call the table something like `user_preference`. Right now, it's just a single user_preference but we'd be able to store other app preferences here as well -- like say if the person wanted to turn the monitor on and off